packages/uswds-core/src/styles/settings/_settings-components.scss

```diff
@@ -110,6 +110,18 @@ $theme-identifier-max-width: "desktop" !default;
 $theme-identifier-primary-link-color: default !default;
 $theme-identifier-secondary-link-color: "base-light" !default;
 
+// In-Page Navigation
+$theme-in-page-nav-background-color: "default" !default;
```

Comment:
Flagging that values with color grades between 20-50 will return values with inaccessible color contrast.

Reply:
hmmm. I'll look into what's happening with `set-link-from-bg()`...

Reply:
At least we do get a good warning:

```
Warning: [In-page-navigation link] Neither the specified preferred link color token (`base-lighter`) nor the fallback link color token (`primary`) have AA contrast on a `blue-30` background. Using `primary`. Please check your source code and project settings.
```

But perhaps we should give folks a setting for link color...

Reply:
Added a new setting for link color

Reply:
@thisisdano I just updated the settings on uswds-site to include the new setting you added $theme-in-page-nav-link-color:
https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/mm-inclusive-patterns-in-page-navigation/components/in-page-navigation/